### ISSUE TITLE
Zwave freez

### ISSUE THREAD

**Issue by filoor** [athombv/com.greenwavesystems#2]
After Adding Greenwave my zwave freezes must reboot Homey for the second time today.

I also send a log "AC8CE9967D"

![screenshot_20160805-194305]
(https://cloud.githubusercontent.com/assets/3647638/17445589/f1670ee2-5b45-11e6-94e4-990ce4747155.png)

**Comment by jghaanstra** [comment]
I have the same issue. For me it started after I added a second Greenwave Powernode-6 which is currently unreachable by Homey. In my < 0.8.39 setup there was a Greenwave Powernode-1 in between Homey and the unreachable Powernode-6 that routed the signal but this device has not been added to my 0.9.2 setup as this device is not available in the app yet.

Another thing to add, the greyed out buttons flicker. Like it tries to start the zwave network but crashes every time. Nothing in the console however.

**Comment by WeeJeWel** [comment]
It's not frozen, the zwave chip is simply busy polling the Greenwave
device. It's not the fastest device I've seen...

Give it some time :)

**Comment by jghaanstra** [comment]
Shouldnt you be enjoying your holidays by now ...  

I gave it a couple of hours already, that should be long enough. Perhaps polling an unreachable device keeps the zwave chip constantly busy?

**Comment by jghaanstra** [comment]
I think I have confirmed my earlier suspicion. I have added an extra zwave device between Homey and my unreachable Greenwave Powernode 6 and give it some time to built the mesh network. After that the issue with the constantly busy zwave chip was resolved (and I could control the unreachable zwave device as well).

So I still think that polling an unreachable device causes the zwave network chip to become unresponsive.
